README.md

```diff
@@ -1,102 +1,70 @@
 
-If you discover a new issue that is not yet reported, please open the same before working on it. People who discover bugs will be rewarded with the amount of coins specified by the **Bug Bounty Owed** labels. 
+### `npm run build`
 
-Please avoid working on issues that have the **Not Ready** label on them.
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The reward labels will be assigned to issues by the core team members based on certain parameters. If you want to create an issue with a design suggestion, please do it in the [design repository](https://github.com/thenewboston-developers/Design).
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
 
-Every PR must be connected to an issue, so as stated above, before creating a PR, please raise an issue and reference it in the PR.
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-Before starting, add a comment to let others know that you'll be working on that issue.
+### `npm run eject`
 
-It is strictly advised to **create a single PR per issue**. In any case, where two or more issues have the same root cause, you can club them in a single PR, otherwise, refrain from doing so.
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
 
-Also, while working make sure you follow these guidelines:
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
-- [React / JSX Style Guide](https://thenewboston.com/style-guide/react)
-- [CSS / SASS Style Guide](https://thenewboston.com/style-guide/css)
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
-Next steps:
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-1. Fork the project
-2. Create your feature branch
-3. Run `npm run lint` and make sure you have no errors
-4. Commit your changes
-5. Push to the branch
-6. Open a pull request
+## Learn More
 
-For further help, you can check out the [FAQs](https://thenewboston.com/faq).
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-Finally, feel free to shoot any queries or suggestions. Don't hesitate. ;)
+To learn React, check out the [React documentation](https://reactjs.org/).
```

Comment:
Can we have coding guide for contributors for instance : Instruct then in the need to comment on each code block - describing what they have done, maintain a particular coding pattern,  the branch to use etc.

Reply:
I think for the coding guide except for the very commons ones we should have to discuss on the channel and see which ones are the most preferred by the majority personally I have not contributed to a lot of open source projects this is my first one.

Reply:
Ok